trace/doc.go

```diff
@@ -0,0 +1,164 @@
+// Package trace provies an API for initiating and reporting SSF
+// traces and attaching spans to them. Veneur's tracing API also
+// provides an experimental opentracing compatibility layer.
+//
+// The Veneur tracing API is completely independent of the opentracing
+// compatibility layer, with the exception of the function
+// StartSpanFromContext.
```

Comment:
I'd recommend adding a quick clause explaining the exception (so that people don't have to look, since StartSpanFromContext is the function they're most likely to be using, so it sounds like it's a bigger exception than it really is)